wasm-test/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "wasm-test"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+surf = {path = "..", default-features = false, features = ["wasm-client"]}
```

Comment:
Does this work without `features = ["wasm-client"]`? i.e. with implicit `wasm_bindgen`?

Reply:
Wasm-bindgen is useless, I think we can remove it. It only applies to the top level crate, which isn't relevant to a library